README.md

```diff
@@ -265,3 +267,48 @@ Solid Queue has been inspired by [resque](https://github.com/resque/resque) and
 
 ## License
```

Comment:
Shouldn't this be at the end?

Reply:
Oops, @brunoprietog thanks for spotting this! It should totally be at the end 😆